continuous-integration/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+# Get and install dependencies
+echo
+echo '-- Prepare --'
+echo "Running on: $DISTRO $OS"
+cd /cwd
+
+if [[ $DISTRO == "ubuntu" ]]; then
+    apt-get update -qq
+    apt-get -y install build-essential
+    apt-get -y install libmount-dev libpcre3-dev python-dev dh-autoreconf flex bison pkg-config zlib1g-dev libffi-dev
+    apt-get -y install libxml2-utils libgamin-dev
+    apt-get -y install software-properties-common
+    add-apt-repository -y ppa:ricotz/testing
```

Comment:
We might have to make our own PPA for mozjs31 until it's available in Ubuntu, since the mozjs31 in this PPA disables the `Intl` API.